File: app/src/main/java/by/overpass/gather/ui/meeting/chat/users/list/viewholder/BaseViewHolder.java
package by.overpass.gather.ui.meeting.chat.users.list.viewholder;

import android.view.View;

import androidx.annotation.NonNull;
import androidx.recyclerview.widget.RecyclerView;

import by.overpass.gather.ui.meeting.chat.users.list.UsersAdapter;
import by.overpass.gather.ui.meeting.chat.users.model.UserModel;

public abstract class BaseViewHolder extends RecyclerView.ViewHolder {

    public BaseViewHolder(@NonNull View itemView) {
        super(itemView);
    }

    public void bind(UserModel userModel, UsersAdapter.OnItemClickListener listener) {
    }
}

File: app/src/main/java/by/overpass/gather/ui/meeting/chat/users/list/viewholder/BaseViewHolder.kt
package by.overpass.gather.ui.meeting.chat.users.list.viewholder

import android.view.View
import androidx.recyclerview.widget.RecyclerView

import by.overpass.gather.ui.meeting.chat.users.list.UsersAdapter
import by.overpass.gather.ui.meeting.chat.users.model.UserModel

abstract class BaseViewHolder(itemView: View) : RecyclerView.ViewHolder(itemView) {

    open fun bind(userModel: UserModel, listener: UsersAdapter.OnItemClickListener) {}
}